src/computability/language.lean

```diff
@@ -104,7 +106,7 @@ def map (f : α → β) : language α →+* language β :=
 by simp [map, image_image]
 
 lemma star_def_nonempty (l : language α) :
```

Comment:
Could you check that all `language` lemma names now refer to `kstar` not `star`?
```suggestion
lemma kstar_def_nonempty (l : language α) :
```

Reply:
Yup, made sure.